lib/ar_state_machine/configuration.rb

```diff
@@ -0,0 +1,11 @@
+module ARStateMachine
+  class Configuration
+
+    attr_accessor :system_id, :should_log_state_change
+
+    def initialize
+      self.should_log_state_change = true
```

Comment:
Since we have AR in the gemspec... why not default ```should_log_state_change``` this to
```ActiveRecord::Base.connection.table_exists? :state_changes```